### PR TITLE
Docs: update datasources that support alerting

### PR DESCRIPTION
**Pull request by briangann**
**What this PR does / why we need it**:

Updates the list of datasources with alerting support.  Added Oracle/FLUX/ADX

**Which issue(s) this PR fixes**:

none, just a document update

### CODE REVIEW

File: docs/sources/alerting/rules.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ and the conditions that need to be met for the alert to change state and trigger
 
 The alert rules are evaluated in the Grafana backend in a scheduler and query execution engine that is part
 of core Grafana. Only some data sources are supported right now. They include `Graphite`, `Prometheus`, `InfluxDB`, `Elasticsearch`,
-`Stackdriver`, `Cloudwatch`, `Azure Monitor`, `MySQL`, `PostgreSQL`, `MSSQL` and `OpenTSDB`.
+`Stackdriver`, `Cloudwatch`, `Azure Monitor`, `MySQL`, `PostgreSQL`, `MSSQL`, `OpenTSDB`, `Oracle`, `Azure Data Explorer`, and `FLUX`.
```
Comment (marefr) [inline docs/sources/alerting/rules.md]: Regarding FLUX, are you referring to https://github.com/grafana/influxdb-flux-datasource? In that case it doesn't have any backend implementation as far as I know?

**Review by marefr — COMMENTED** [review]
I understand why you suggest these changes, but doesn't feel like this would scale if we're planning to add every possible backend datasource implementation to this list. The plugin json have a backend property so one idea would be to expose this on grafana.com somehow like in the plugin list and in the plugin info/readme. Thoughts?

Please have a look at my comment as well which I don't understand.

File: docs/sources/alerting/rules.md (hunk 1 above)
Comment (briangann) [reply docs/sources/alerting/rules.md]: my mistake on FLUX, definitely has no backend

**Comment by briangann** [comment]
a more automated method to keep this list up to date would be great

**Review by ryantxu — APPROVED** [review]
@marefr -- I totally agree this is not a scaleable process, and something we need to revisit.  However, this change is our best easiest option today.

**Pull request merged**